cypress/integration/default/images.spec.ts

```diff
@@ -17,4 +17,15 @@ describe('next/images', () => {
     cy.visit('/image')
     cy.findByRole('img', { name: /shiba inu dog looks through a window/i })
   })
-})
\ No newline at end of file
+
```

Comment:
It would be good to have a test for an image _failing_ to load when it doesn't match the pattern